cli/command/secret/create.go

```diff
@@ -45,7 +45,7 @@ func runSecretCreate(dockerCli *command.DockerCli, options createOptions) error
 
 	secretData, err := ioutil.ReadAll(os.Stdin)
 	if err != nil {
-		return fmt.Errorf("Error reading content from STDIN: %v", err)
+		return fmt.Errorf("error reading content from STDIN: %v", err)
```

Comment:
Yeah that the tricky part… `golang` best pratices and *linting* tools says it should be lowercase. But we print it as is so that kinda feels weird. (we have this at some other places). I'm not sure what is the best thing to do here (I'm fine with lowercase errors but meh :angel: ).